feature-libs/cart/main/root/cart-root.module.ts

```diff
@@ -48,7 +48,7 @@ export function defaultCartComponentsConfig() {
       },
     ]),
   ],
-  exports: [MiniCartModule],
+  exports: [MiniCartModule, AddToCartModule],
```

Comment:
`imports` or `exports`?

Reply:
Import. This should have been `imports` for `MiniCartModule` to begin with. (my bad)